.github/workflows/ci.yaml

```diff
@@ -1,11 +1,27 @@
 # reusable workflow triggered by other actions
-name: CI
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 
 on:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * TUE'
+
+  # Triggered on push by .github/workflows/release.yaml
   workflow_call:
-    secrets:
-      CHARMCRAFT_CREDENTIALS:
-        required: true
+    outputs:
+      artifact-prefix:
+        description: build_charm.yaml `artifact-prefix` output
+        value: ${{ jobs.build.outputs.artifact-prefix }}
+      charm-paths:
+        description: paths for all charms in this repo
+        value: ${{ jobs.get-charm-paths-channel.outputs.charm-paths }}
+      channel:
+        description: Charmhub channel the charms are released to
+        value: ${{ jobs.get-charm-paths-channel.outputs.charm-channel }}
 
 jobs:
   lib-check:
```

Comment:
suggestion(non-blocking): this job can be updated to use `get-charm-paths` output to populate the matrix instead of hard-coding it

Reply:
How can I say no to such a nice suggestion!

14d8715